processing/art/fireworks/fireworks.js

```diff
@@ -0,0 +1,250 @@
+// Forked from https://github.com/andrewdcampbell/jsfireworks/blob/master/sketch.js
+// const SHELLTYPES = ['simple', 'split', 'burst', 'double',
+//                     'mega', 'writer', 'pent', 'comet'];
+const SHELLTYPES = ['mega']
+const GRAVITY = 0.2;
+var PAUSED = true;
+
+var CANVAS_WIDTH = 3000;
+var CANVAS_HEIGHT = 2400;
```

Comment:
I used to do this as well, but p5.js exposes https://p5js.org/reference/#/p5/width and https://p5js.org/reference/#/p5/height that are initialized after calling createCanvas

Reply:
https://github.com/calebmeyer/generative-art/commit/7da3b357ec2ca16aa3206016be5a0e48b0c0ba9a